graphene-reader/src/main/java/net/iponweb/disthene/reader/handler/RenderHandler.java

```diff
@@ -91,32 +95,32 @@ public FullHttpResponse handle(RenderParameter parameters) throws ParameterParsi
         }
 
         logger.info("targets : " + targets);
-        FullHttpResponse response;
+        ResponseEntity<?> response;
         try {
-            response = timeLimiter.callWithTimeout(new Callable<FullHttpResponse>() {
+            response = timeLimiter.callWithTimeout(new Callable<ResponseEntity>() {
                 @Override
-                public FullHttpResponse call() throws EvaluationException, LogarithmicScaleNotAllowed {
+                public ResponseEntity<?> call() throws EvaluationException, LogarithmicScaleNotAllowed {
                     return handleInternal(targets, parameters);
                 }
             }, readerConfiguration.getRequestTimeout(), TimeUnit.SECONDS, true);
         } catch (UncheckedTimeoutException e) {
             logger.debug("Request timed out: " + parameters);
             statsService.incTimedOutRequests(parameters.getTenant());
-            response = new DefaultFullHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.REQUEST_ENTITY_TOO_LARGE);
+            response = ResponseEntity.status(HttpStatus.REQUEST_ENTITY_TOO_LARGE).build();
         } catch (EvaluationException | LogarithmicScaleNotAllowed e) {
             throw e;
         } catch (Exception e) {
             logger.error(e);
-            response = new DefaultFullHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.INTERNAL_SERVER_ERROR);
+            response = ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
         }
 
         timer.stop();
         logger.debug("Request took " + timer.elapsed(TimeUnit.MILLISECONDS) + " milliseconds (" + parameters + ")");
 
-        logger.info("Response status : " + response.getStatus() + " Response Body : " + response.content().toString());
+        logger.info("Response status : " + response.getStatusCode());
```

Comment:
I think it's wise not to leave response body in INFO log.

Reply:
That's right. now graphene exposes Unnecessarily a lot of response body. 
LGTM :)